haystack/database/base.py

```diff
@@ -1,15 +1,17 @@
 from abc import abstractmethod, ABC
-from typing import Any, Optional, Dict, List
+from typing import Any, Optional, Dict, List, Union
 from uuid import uuid4
 
+import numpy as np
+
 
 class Document:
     def __init__(self, text: str,
                  id: str = None,
                  query_score: Optional[float] = None,
                  question: Optional[str] = None,
                  meta: Dict[str, Any] = None,
-                 embedding: Optional[List[float]] = None):
+                 embedding: Optional[np.array] = None):
```

Comment:
does np.array also hold for embeddings coming back from ES?

Reply:
Yes, it's now changed to `np.array` across Haystack.